iterfzf/__init__.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 import subprocess
 import sys
-from typing import AnyStr, Iterable, Literal, Optional
+from typing import AnyStr, Iterable, Literal, Optional, Dict
```

Comment:
Could you list them in the lexicographical order?

Reply:
Yes, put it last cause it was 'concrete'.
It will change to AnyStr, Iterable, Literal, Mapping, Optional now.